models/brats_mri_generative_diffusion/docs/README.md

```diff
@@ -0,0 +1,165 @@
+# Model Overview
+A pre-trained model for volumetric (3D) Brats MRI 3D Latent Diffusion Generative Model.
+
+This model is trained on BraTS 2016 and 2017 data from [Medical Decathlon](http://medicaldecathlon.com/), using the Latent diffusion model [1].
+
+![model workflow](https://developer.download.nvidia.com/assets/Clara/Images/monai_brain_image_gen_ldm3d_network.png)
+
+This model is a generator for creating images like the Flair MRIs based on BraTS 2016 and 2017 data. It was trained as a 3d latent diffusion model and accepts Gaussian random noise as inputs to produce an image output. The `train_autoencoder.json` file describes the training process of the variational autoencoder with GAN loss. The `train_diffusion.json` file describes the training process of the 3D latent diffusion model.
+
+**This is a demonstration network meant to just show the training process for this sort of network with MONAI. To achieve better performance, users need to use larger dataset like [Brats 2021](https://www.synapse.org/#!Synapse:syn25829067/wiki/610865) and have GPU with memory larger than 32G to enable larger networks and attention layers.**
+
+## Install the dependency of MONAI generative models
+[MONAI generative models](https://github.com/Project-MONAI/GenerativeModels) can be installed by
+```
+pip install lpips==0.1.4
+pip install git+https://github.com/Project-MONAI/GenerativeModels.git@0.2.0
+```
+
+Or we can install it from source
+```
+pip install lpips==0.1.4
+git clone https://github.com/Project-MONAI/GenerativeModels.git
+cd GenerativeModels/
+python setup.py install
+cd ..
+```
+
+## Downloading the Dataset
+The training data is Brats 2016 and 2017 from [Medical Decathlon](http://medicaldecathlon.com/).
+
+Target: image generatiion
+Task: Synthesis
+Modality: MRI
+Size: 388 3D volumes (1 channel used)
+
+The dataset can be downloaded automatically at the beggining of training.
+
+## Training configuration
+If user has GPU memory smaller than 32G, then please decrease the `"train_batch_size"` in `configs/train_autoencoder.json` and `configs/train_diffusion.json`.
+
+### Training configuration of autoencoder
+The training of autoencoder was performed with the following:
+
+- GPU: at least 32GB GPU memory
+- Actual Model Input: 112 x 128 x 80
+- AMP: False
+- Optimizer: Adam
+- Learning Rate: 1e-5
+- Loss: L1 loss, perceptual loss, KL divergence loss, adversianl loss, GAN BCE loss
+
+#### Input
+1 channel 3D MRI Flair patches
+
+#### Output
+- 1 channel 3D MRI reconstructed patches
+- 8 channel mean of latent features
+- 8 channel standard deviation of latent features
+
+### Training configuration of diffusion model
```

Comment:
### Training Configuration of Diffusion Model
The latent diffusion model was trained using the following configuration: